src/main/java/com/iexec/sms/secret/app/owner/AppDeveloperTeeTaskRuntimeSecretController.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms.secret.app.owner;
+
+import com.iexec.sms.authorization.AuthorizationService;
+import com.iexec.sms.secret.SecretUtils;
+import com.iexec.sms.secret.app.DeployedObjectType;
+import com.iexec.sms.secret.app.OwnerRole;
+import com.iexec.sms.secret.app.TeeTaskRuntimeSecretCountService;
+import com.iexec.sms.secret.app.TeeTaskRuntimeSecretService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@CrossOrigin
+@RestController
+@RequestMapping("/apps")
+public class AppDeveloperTeeTaskRuntimeSecretController {
```

Comment:
```suggestion
public class AppRuntimeSecretController {
```
(same in methods of this class, TeeTaskRuntime -> AppRuntime)
(since controller dedicated to secrets of AppRuntime)

We might add /requesters/[..] AppRuntime endpoint inside

Reply:
Alright! I've moved this controller outside of the `owner` package as it is not anymore only relating to app developer's secrets.